website/tests/test_website_forms.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+from ..forms import EditFixedSectionForm, AddEditPageSectionForm, \
+    AddEditNewsPostForm, AddEditPublicationForm, AddEditCarouselImageForm
+from django.utils import timezone
+
+# Todo: test POST and submit button
+
+
+def test_init_form():
+    assert EditFixedSectionForm()
+    assert AddEditPageSectionForm()
+    assert AddEditNewsPostForm()
+    assert AddEditPublicationForm()
+    assert AddEditCarouselImageForm()  # entry=self.entry)
+
+
+@pytest.mark.django_db(transaction=False)
+def test_edit_fixed_section_form():
+
+    # Test Valid Data
+    form = EditFixedSectionForm({
+        'title': 'My test',
+        'body_markdown': 'This is a test!',
+    })
+    assert form.is_valid()
+
+    section = form.save()
+    assert section.title == 'My test'
```

Comment:
I get some doubt too when I created form's tests. Do you know any other way to test forms or should we test them only through view ?